src/content_sao.cpp

```diff
@@ -366,7 +367,7 @@ std::string LuaEntitySAO::getClientInitializationData(u16 protocol_version)
 		writeU8(os, 4 + m_bone_position.size()); // number of messages stuffed in here
 		os<<serializeLongString(getPropertyPacket()); // message 1
 		os<<serializeLongString(gob_cmd_update_armor_groups(m_armor_groups)); // 2
-		os<<serializeLongString(gob_cmd_update_animation(m_animation_range, m_animation_speed, m_animation_blend)); // 3
+		os<<serializeLongString(gob_cmd_update_animation(m_animation_range, m_animation_speed, m_animation_blend, m_animation_loop)); // 3
```

Comment:
I'm not sure exactly how to break this into multiple lines, nor if I should do it in this patch. The line is actually as long as others above / below it, so I'd need to break all of them. And if I started doing this, I'd fill the patch with a lot of code style cleanups rather than focusing on the feature. I'd suggest someone else does this separately afterward, for all lines which are considered too long... although they seem fine to me personally.